modules/http/src/main/scala/higherkindness/mu/http/Utils.scala

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2017-2019 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package higherkindness.mu.http
+
+import cats.ApplicativeError
+import cats.effect._
+import cats.implicits._
+import fs2.interop.reactivestreams._
+import fs2.{RaiseThrowable, Stream}
+import io.grpc.Status.Code._
+import org.typelevel.jawn.ParseException
+import io.circe._
+import io.circe.generic.auto._
+import io.circe.jawn.CirceSupportParser.facade
+import io.circe.syntax._
+import io.grpc.{Status => _, _}
+import jawnfs2._
+import monix.execution._
+import monix.reactive.Observable
+import org.http4s._
+import org.http4s.dsl.Http4sDsl
+import scala.concurrent.ExecutionContext
+import scala.util.control.NoStackTrace
+
+object Utils {
+
+  implicit class MessageOps[F[_]](val message: Message[F]) extends AnyVal {
```

Comment:
I'd make the value private.  Otherwise you're leaking a `message` method on all `Message[F]`:

```
val x: Message[F] = ???
x.message.message.message.message
```

```suggestion
  implicit class MessageOps[F[_]](private val message: Message[F]) extends AnyVal {
```

Reply:
Done at https://github.com/higherkindness/mu/commit/c922e0f1da911b7a61752907dc882e8522eae495